client/src/Pages/Home.js

```diff
@@ -37,7 +42,14 @@ const Home = () => {
             >
             Learn React { isTabletOrMobile && <p>Yo we on mobile</p>}
             </a>
+            <SearchButtonDiv>
+                <SearchButton onClick={() => toSearch()}>
```

Comment:
Instead of using your own custom button, I would import a button from Material UI, since this is the standard for our project. Here's a link that shows you how to use that: [Material UI Button](https://mui.com/components/buttons/)

Reply:
Got it, just changed

Reply:
Oh wait, think this is the only place where Material UI is used for button; is this okay?